api/kessel/inventory/v1beta1/policies_service.proto

```diff
@@ -15,10 +15,7 @@ message CreatePolicyRequest {
   Policy policy = 1;
 }
 
-message CreatePolicyResponse {
-  // The policy created in Kessel Asset Inventory
-  Policy policy = 1;
-}
+message CreatePolicyResponse {}
```

Comment:
About the requirement of not returning anything in the create requests.
Wouldn't this be an issue for the caller if they want to do any subsequent call with the newly created resource?
e.g. getting the id of the newly created resource for any following up process to initialize or prepare anything else?